### PR TITLE
Localization for 2.38

### PR DESCRIPTION
**Pull request by mikeger**
Crowdin decided to reformat the xml files, but they are still valid.

### CODE REVIEW

File: Wire-iOS/Resources/fi.lproj/Localizable.strings [hunk 1]
```diff
@@ -27,7 +27,7 @@
 "peoplepicker.search_placeholder" = "Etsi nimellä tai käyttäjänimellä";
 "peoplepicker.header.top_people" = "Eniten käytetyt kontaktit";
 "peoplepicker.button.create_conversation" = "Luo ryhmä";
-"peoplepicker.button.add_to_conversation" = "Lisää";
+"peoplepicker.button.add_to_conversation" = "Add People to Group";
```
Comment (vytis) [inline Wire-iOS/Resources/fi.lproj/Localizable.strings]: This seems off

Comment (mikeger) [inline Wire-iOS/Resources/fi.lproj/Localizable.strings]: The value was changed in English, then Crowdin also replaces all other languages values.

File: Wire-iOS/Resources/nl.lproj/Localizable.strings [hunk 2]
```diff
@@ -27,7 +27,7 @@
 "peoplepicker.search_placeholder" = "Zoek op naam of gebruikersnaam";
 "peoplepicker.header.top_people" = "Favoriete contacten";
 "peoplepicker.button.create_conversation" = "Groep creëren";
-"peoplepicker.button.add_to_conversation" = "Toevoegen";
+"peoplepicker.button.add_to_conversation" = "Add People to Group";
```
Comment (vytis) [inline Wire-iOS/Resources/nl.lproj/Localizable.strings]: Same here...

File: Wire-iOS/Resources/fi.lproj/Localizable.strings (hunk 1 above)
Comment (vytis) [inline Wire-iOS/Resources/fi.lproj/Localizable.strings]: Makes sense

**Pull request merged**